readthedocs/builds/querysets.py

```diff
@@ -91,9 +91,9 @@ def _add_user_repos(self, queryset, user=None):
         if user.has_perm('builds.view_version'):
```

Comment:
btw, we don't set this in any part of the code, so I guess this can only be done in the db, but again, I don't think we do that or want to do that.